### ISSUE TITLE
chore: Latest pip 24.1 will fail to install celery

### ISSUE THREAD

**Issue by disaac** [lardbit/nefarious#294]
During some testing on my fork I noticed the Dockerfile for nefarious doesnt specify the version of pip and will fail on build with the following:

```text
#13 17.63     pytz (>dev)
#13 17.63          ~^
#13 17.63 Please use pip<24.1 if you need to use this version.
```
This is due to [pypa/pip#12063](https://github.com/pypa/pip/issues/12063)

To address this temporarily on my build I pinned the pip version in the Dockerfile. Ideally a newer celery package will be used that utilizes the updated specifier for >24.0 pip

```bash
    && /env/bin/pip install --force-reinstall -v "pip==24.0" \
```

**Comment by disaac** [comment]
@lardbit worked around with fix in PR #297

**Comment by lardbit** [comment]
Interesting new detail about pinning the specific versions.  Good catch.

**Comment by lardbit** [comment]
the celery instances uses the same docker image so your fix will solve it too

**Comment by lardbit** [comment]
Merged in #297
